smriprep/workflows/outputs.py

```diff
@@ -731,6 +731,8 @@ def init_ds_surfaces_wf(
                 ds_surf.inputs.desc = 'reg'
                 if surf == 'sphere_reg_fsLR':
                     ds_surf.inputs.space = 'fsLR'
+                elif surf == 'sphere_reg':
+                    ds_surf.inputs.space = 'fsaverage'
```

Comment:
At this point the nested `if`/`else` is starting to look convoluted. What about setting the values for `sphere.reg` and then overriding for specific known variants?

```python
if surf.startswith('sphere_reg'):
    ds_surf.inputs.space, ds.surf.inputs.desc = 'fsaverage', 'reg'  # Default
    if surf == 'sphere_reg_fsLR':
        ds_surf.inputs.space = 'fsLR'
    elif surf == 'sphere_reg_msm':
        ds_surf.inputs.space, ds.surf.inputs.desc = 'fsLR', 'msmsulc'
```

Reply:
That works for me. I'll add it in